modules/react/popup/lib/hooks/useCloseOnOutsideClick.ts

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import {PopupStack} from '@workday/canvas-kit-popup-stack';
+import {PopupModel} from '../usePopupModel';
+
+/**
+ * Registers global listener for all clicks. It will only call the PopupModel's `hide` event if the
```

Comment:
Would be good to add a note about when to use this vs. `useAlwaysCloseOnOutsideClick`